src/test/Testcafe/testRunner.js

```diff
@@ -17,7 +17,7 @@ createTestCafe()
       .run({
         skipJsErrors: true,
         skipUncaughtErrors: true,
-        quarantineMode: true,
+        quarantineMode: false,
```

Comment:
retry on failed test cases are stopped . Should the quarantineMode be set back to true ?

Reply:
Fixed